policy/approval/approve_test.go

```diff
@@ -200,11 +216,62 @@ func TestIsApproved(t *testing.T) {
 		assertApproved(t, prctx, r, "Approved by comment-approver, review-approver")
 	})
 
-	t.Run("contributorsCanApprove", func(t *testing.T) {
+	t.Run("contributorsIncludingAuthorCanApprove", func(t *testing.T) {
 		prctx := basePullContext()
 		r := &Rule{
 			Options: Options{
 				AllowContributor: true,
+				AllowAuthor:      false,
+			},
+			Requires: Requires{
+				Count: 1,
+				Actors: common.Actors{
+					Organizations: []string{"everyone"},
+				},
+			},
+		}
+		assertApproved(t, prctx, r, "Approved by comment-approver, mhaypenny, contributor-author, contributor-committer, review-approver")
+	})
+
+	t.Run("contributorsExcludingAuthorCanApprove", func(t *testing.T) {
+		prctx := basePullContext()
+		r := &Rule{
+			Options: Options{
+				AllowNonAuthorContributor: true,
+			},
+			Requires: Requires{
+				Count: 1,
+				Actors: common.Actors{
+					Organizations: []string{"everyone"},
+				},
+			},
+		}
+		assertApproved(t, prctx, r, "Approved by comment-approver, review-approver")
```

Comment:
I believe this should should approvals by `contributor-author` and `contributor-committer`. Both are non-author contributors on the PR who left approvals.

Reply:
oh yea forgot about the +1 default